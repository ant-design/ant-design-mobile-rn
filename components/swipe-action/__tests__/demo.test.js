import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('swipe-action')
