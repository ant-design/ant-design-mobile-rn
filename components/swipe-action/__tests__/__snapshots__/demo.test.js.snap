// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/swipe-action/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "paddingTop": 30,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderTopColor": "#dddddd",
          "borderTopWidth": 0.5,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#dbddde",
              "overflow": "hidden",
            },
            Object {
              "backgroundColor": "transparent",
            },
          ]
        }
      >
        <View
          onLayout={[Function]}
          onMoveShouldSetResponder={[Function]}
          onMoveShouldSetResponderCapture={[Function]}
          onResponderEnd={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderReject={[Function]}
          onResponderRelease={[Function]}
          onResponderStart={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          onStartShouldSetResponderCapture={[Function]}
          style={
            Array [
              Object {},
              Object {
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              },
            ]
          }
        >
          <
            activeOpacity={1}
            delayLongPress={500}
            onLongPress={[Function]}
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#ffffff",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "paddingLeft": 15,
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "borderBottomColor": "#dddddd",
                      "borderBottomWidth": 0.5,
                      "flex": 1,
                      "flexDirection": "row",
                      "minHeight": 44,
                      "paddingRight": 15,
                      "paddingVertical": 6,
                    },
                    false,
                    false,
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "flexDirection": "column",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 17,
                          "textAlignVertical": "center",
                        },
                      ]
                    }
                  >
                    Simple example: left and right buttons
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "flexDirection": "column",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Array [
                        Object {
                          "color": "#888888",
                          "fontSize": 17,
                          "textAlign": "right",
                          "textAlignVertical": "center",
                        },
                      ]
                    }
                  >
                    extra content
                  </Text>
                </View>
              </View>
            </View>
          </>
        </View>
        <View />
        <View />
      </View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#ffffff",
              "borderBottomColor": "#dddddd",
              "borderBottomWidth": 0.5,
              "bottom": 0,
              "height": 1,
              "left": 0,
              "position": "absolute",
              "right": 0,
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;
